cpp/src/gandiva/precompiled/extended_math_ops_test.cc

```diff
@@ -87,6 +87,19 @@ TEST(TestExtendedMathOps, TestLogWithBase) {
   EXPECT_EQ(context1.has_error(), false);
 }
 
+TEST(TestExtendedMathOps, TestRound) {
+  EXPECT_EQ(round_int32(7589, -1), 7590);
```

Comment:
these tests are a little hard to read but I don't think any of them actually check rounding at the boundary (digit = 5)

Reply:
EXPECT_EQ(round_int32(8612, -5), 0) tests the boundary conditions. For 5 decimal places, the num before rounding is 0.08612 which rounds down to 0.

Reply:
what I mean is I don't think anything here test for the addition of .5, I think there are currently other values for the rounding addition that would allow these tests to pass